### PR TITLE
meta: note that the repo and process has been archived

### PR DESCRIPTION
**Pull request by jasnell**
See https://github.com/nodejs/TSC/issues/335

Ping @nodejs/tsc

### CODE REVIEW

**Review by Trott — APPROVED** [review]
LGTM. Not sure people are going to understand that "decided to archive the node-eps repository and process" means "don't use this". (The word I have questions about is "archive". Might be better as "discontinue" for clarity?) But I'm nit-picking. We can always tweak the wording if it actually ever does cause confusion.

**Comment by unional** [comment]
What is the preferred mechanism for discussing proposals such as https://github.com/nodejs/node-eps/pull/60 moving forward?

It would be good to add some actionable steps in the message. 🌷

**Comment by jasnell** [comment]
Pull requests against master or working in other forks the way we've done with things like http2, Node-ChakraCore, etc

**Comment by unional** [comment]
> Pull requests against master or working in other forks the way we've done with things like http2, Node-ChakraCore, etc

On https://github.com/nodejs/node directly?

**Comment by jasnell** [comment]
Yes. Or in working repos such as nodejs/http2

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Node.js Enhancement Proposals
 
+**NOTE**: The Node.js TSC has [recently](https://github.com/nodejs/TSC/issues/335)
+decided to archive the node-eps repository and process. See https://github.com/nodejs/TSC/issues/335
```
Comment (thefourtheye) [inline README.md:4]: Nit: Linking the same issue twice in two consecutive lines kinda feels redundant.

**Review by thefourtheye — APPROVED** [review]

**Comment by thefourtheye** [comment]
Perhaps we should list all the repos under our roof and explain what they are for, in our main README.md.

**Comment by bengl** [comment]
Can `(Archived/Inactive)` be added to this project's description? That would be helpful in the several contexts in which the description is shown.

**Comment by Trott** [comment]
Since GitHub offers genuine read-only archiving now, I *think* this is obsolete. Feel free to re-open or comment if I'm wrong.

**Pull request closed without merging**